inlineplz/linters/__init__.py

```diff
@@ -277,13 +277,20 @@ def run_command(command, log_on_fail=False, log_all=False):
     shell = False
     if os.name == 'nt':
         shell = True
+    local_env = {}
+    safe_envvars = [
+        b'PATH', b'PYTHONPATH', b'GOPATH', b'SystemRoot',
```

Comment:
Looks like this could balloon into many language + tool aware exceptions. Can I get the context on what this is trying to solve?